polkadot-parachains/westmint/src/weights/paritydb_weights.rs

```diff
@@ -0,0 +1,63 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub mod constants {
```

Comment:
I don't understand all these extra `constants` modules. Why do we need them?

Reply:
We don't, it is just a naming convention from Substrate.  
Does not really makes tbh, so I will remove them from the weight templates.